### PR TITLE
Add Contributor Covenant Code of Conduct

### PR DESCRIPTION
**Pull request by rawrmonstar**
Added CoC per @ebrillhart.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,3 +7,78 @@ Before submitting a PR / pulishing a release make sure to:
 * Update and commit changes to README.md with: `npm run builder:build`
 
 Thanks for helping out!
```
Comment (ryan-roemer) [inline CONTRIBUTING.md]: Can you add:

```
## Pull Requests
```

as a similar heading above the section above that begins with `Before submitting`?

**Comment by ebrillhart** [comment]
LGTM!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,86 @@
 Contributing
 ============
 
-Before submitting a PR / pulishing a release make sure to:
```
Comment (ryan-roemer) [inline CONTRIBUTING.md]: Nice catch on `pulishing` 😄

**Comment by ryan-roemer** [comment]
LGTM.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rawrmonstar) [inline CONTRIBUTING.md]: I had to atone for the typo on the branch name task-addCodeOfCond**e**ct 😐

**Pull request merged**